src/toDo.js

```diff
@@ -0,0 +1,40 @@
+import threedot from './assets/3-dot.svg';
+
+function toDo() {
+  const tasks = [
+    {
+      description: 'Task 1',
+      completed: false,
+      index: 0,
+    },
+    {
+      description: 'Task 2',
+      completed: true,
+      index: 1,
+    },
+    {
+      description: 'Task 3',
+      completed: false,
+      index: 2,
+    },
+  ];
+
+  // to do list function
+  function toDoList() {
+    const listContainer = document.querySelector('.to-do-list');
+
+    tasks.forEach((task) => {
+      // Create list item element
+      const listItem = document.createElement('li');
+      listItem.classList.add('list-items');
+      listItem.innerHTML = `<input type='checkbox' id='list-checkbox' name='checkbox'><p class='list-paragraph'>${task.description}</p><img src='${threedot}'  alt='#' class='list-img'>`;
```

Comment:
- [ ] Nice one dynamically rendering the tasks, but kindly note that the `checkbox` or in your case the  `checked` property of the checkbox  should be rendered dynamically also with respect to the `completed` in each objects, meaning it should be checked on `completed: true` and unchecked on completed false...

You could do something like. `<input class='checkbox' type="checkbox" ${task.completed && 'checked'}/>` or more clearer with ternary operator `<input class='checkbox' type="checkbox" ${elem.completed? 'checked' : ''}/>`

the `checked` property is like the `required` property, but what it does is that it triggers the checked icon of the checkbox, when it is present.